lib/koans/10_map_sets.ex

```diff
@@ -0,0 +1,62 @@
+defmodule MapSets do
+  use Koans
+
+  @intro "My name is Set, MapSet."
+
+  @set MapSet.new([1, 2, 3, 4, 5])
+
+  koan "I am merely another collection but, you can perform some operations on me" do
+    new_set = MapSet.new(@set, fn x -> 3 * x end)
```

Comment:
Now that I think about it, I think you are right @iamvery.

how about we add a very first koan that shows that 'its kinda like list'?
Create a MapSet and then use any fun `Enum` function on it like you would with a list.
And then show duplication and then proceed to any of the fun MapSet specific APIs?

@msurekci thanks for sticking with us ❤️